cli/tests/integrations/test_package.py

```diff
@@ -75,8 +75,14 @@ def test_repo_list():
         ).format(**UNIVERSE_TEST_REPOS),
         'utf-8'
     )
+
     assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
 
+    # test again, but override the dcos_url with a cosmos_url config
+    dcos_url = config.get_config_val("core.dcos_url")
+    with update_config('package.cosmos_url', dcos_url, env):
+        assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
+
```

Comment:
@tamarrow Sorry for not being clear. Let's create another test. E.g. `test_package_cosmos_url`. Thanks!